src/style.css

```
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 800;
```

```suggestion
  width: 100%;
  height: 70px;
  padding-left: 62px;
  
  line-height: 22px;
  font-size: 16px;
  font-family: "Avenir", sans-serif;
  color: #3d4e61;
  font-weight: 800;
  
  background: #fff;
  border: 1px solid #e1e7ed;
  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
  border-radius: 4px;
```
better group rules by theme, like {
  position

  block model

  typography

  appearance

  other
}

---

Understood, thanks!